src/main/java/mate/jdbc/service/DriverService.java

```diff
@@ -0,0 +1,17 @@
+package mate.jdbc.service;
+
+import java.util.List;
+import java.util.Optional;
+import mate.jdbc.model.Driver;
+
+public interface DriverService {
+    Driver create(Driver driver);
+
+    Optional<Driver> get(Long id);
+

```

Comment:
![image](https://user-images.githubusercontent.com/115244741/227371579-d21c0185-6423-4d44-ba3d-20601e0387cc.png)
we need return object here